<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="NC003" name="testcases.dashboard.practice.practiceProvider.rolling.TC_PracticeProvidersPatientDrillDownCountR" tests="1" failures="0" timestamp="14 Jun 2018 09:18:44 GMT" time="0.000" errors="1">
  <testcase name="TC_PPPatientDrillDown" time="0.000" classname="testcases.dashboard.practice.practiceProvider.rolling.TC_PracticeProvidersPatientDrillDownCountR">
    <error type="org.testng.TestNGException" message="
Data Provider public java.lang.Object[][] testcases.dashboard.practice.practiceProvider.rolling.TC_PracticeProvidersPatientDrillDownCountR.dataProvider() throws java.io.IOException,org.apache.poi.openxml4j.exceptions.InvalidFormatException returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.lang.Object[][] testcases.dashboard.practice.practiceProvider.rolling.TC_PracticeProvidersPatientDrillDownCountR.dataProvider() throws java.io.IOException,org.apache.poi.openxml4j.exceptions.InvalidFormatException returned a null value
at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:133)
at org.testng.internal.Parameters.handleParameters(Parameters.java:706)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at TestNGLoader.Loader.main(Loader.java:60)
]]>
    </error>
  </testcase> <!-- TC_PPPatientDrillDown -->
</testsuite> <!-- testcases.dashboard.practice.practiceProvider.rolling.TC_PracticeProvidersPatientDrillDownCountR -->
